ruby/bps-kafka/spec/bps/kafka_spec.rb

```diff
@@ -5,15 +5,6 @@ def kafka_addrs
   ENV.fetch('KAFKA_ADDRS', '127.0.0.1:9092').split(',').freeze
```

Comment:
same, remove this method and move it as a `let` helper into the spec itself

Reply:
I'm afraid these let-s won't properly work:

```
  `nats_servers` is not available on an example group (e.g. a `describe` or `context` block). It is only available from within individual examples (e.g. `it` blocks) or from constructs that run in the scope of an example (e.g. `before`, `let`, etc).
```

Like, we do `it_behaves_like 'publisher', url: "HERE-LETS-ARE-INTERPOLATED"`.

Reply:
I think, we can do `let(:url)` instead of passing it in features, let me try this.